app/views/admin/dashboard/index.html.erb

```diff
@@ -1,27 +1,44 @@
-</div>
-
-<div class="dead-companies">
-  <% @dashboard_presenter.dead_companies.each do |company| %>
-    <%= link_to company.organization, admin_company_path(company.id) %>
-  <% end %>
-</div>
+<div class="container">
+  <nav class="navbar navbar-default">
+    <div class="navbar-header">
+      <button class="navbar-toggle" type="button" data-toggle="collapse" data-target=".js-navbar-collapse">
+        <span class="sr-only">Toggle navigation</span>
+        </button>
+        <a class="navbar-brand" href="#">Admin Dashboard</a>
+        <a class="navbar-brand" href="/admin/companies">Companies</a>
+        </div>
+        <div class="collapse navbar-collapse js-navbar-collapse">
+          <ul class="nav navbar-nav">
+            <li class="dropdown mega-dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown">Manage Content <span class="glyphicon glyphicon-chevron-down pull-right"></span></a>
+              <ul class="dropdown-menu mega-dropdown-menu row">
+                <li class="col-sm-3">
+                  <ul>
+                    <li class="dropdown-header">Content Manager</li>
+                    <li class="divider"></li>
+                    <li><a href="#">View As User <span class="glyphicon glyphicon-chevron-down pull-right"></span></a></li>
+                  </ul>
+                    </li>
+                    <li class="col-sm-4">
+                  <ul>
+                    <li class="dropdown-header manage-static-link"><a href="#">Manage Static Content <span class="glyphicon glyphicon-list"></span></a></li>
+                    <li class="dropdown-header manage-pdf"><a href="#">Manage PDF Files <span class="glyphicon glyphicon-file"></span></a></li>
+                    <li class="dropdown-header manage-video"><a href="#">Manage Video Files <span class="glyphicon glyphicon-film"></span></a></li>
+                    <li class="dropdown-header manage-map"><a href="#">Manage Map <span class="glyphicon glyphicon-pushpin"></span></a></li>
+                  </ul>
+                    </li>
+                    <li class="col-sm-4">
+                  <ul>
+                    <li class="dropdown-header manage-tools"><a href="#">Tools/Resources <span class="glyphicon glyphicon-wrench"></a></li>
+                    <li class="dropdown-header manage-learn"><a href="#">Learn More <span class="glyphicon glyphicon-education"></span></a></li>
+                    <li class="dropdown-header manage-sign-up"><a href="#">Company Sign Up <span class="glyphicon glyphicon-pencil"></span></a></li>
+                    <li class="dropdown-header manage-home"><a href="#">Home <span class="glyphicon glyphicon-home"></span></a></li>
+                  </ul>
+                    </li>
+                  </ul>
+                    </li>
+                  </ul>
+                  <a class="nav navbar-right" href="#">Logout</a>
+                </div><!-- /.nav-collapse -->
+              </nav>
+            </div>
```

Comment:
Looks good! Could you tell me about the location of the view though? It looks like the dashboard controller does nothing anymore, and that most of the action is on the admin companies controller. Should this be a partial available on all the admin pages? I'm just curious what else the user would see if they visit this page, since it just looks like a navbar. I'm fine with the navbar, but think it seems confusing from a user perspective to have a page with just a navbar on it.